pygmt/datasets/earth_relief.py

```diff
@@ -23,9 +23,10 @@ def load_earth_relief(resolution="60m"):
     Parameters
     ----------
     resolution : str
-        The grid resolution. The suffix ``m`` and ``s`` stand for arc-minute
-        and arc-second. It can be ``'60m'``, ``'30m'``, ``'10m'``, ``'05m'``,
-        ``'02m'``, ``'01m'``, ``'30s'`` or ``'15s'``.
+        The grid resolution. The suffix ``d``, ``m`` and ``s`` stand for
+        arc-degree, arc-minute and arc-second. It can be ``'01d'``, ``'60m'``,
+        ``'30m'``, ``'10m'``, ``'05m'``, ``'02m'``, ``'01m'``, ``'30s'``
+        or ``'15s'``.
```

Comment:
```suggestion
        arc-degree, arc-minute and arc-second. It can be ``'01d'``, ``'30m'``,
        ``'10m'``, ``'05m'``, ``'02m'``, ``'01m'``, ``'30s'`` or ``'15s'``.
```

Can probably remove the "60m" here. Also will need to add 3 and 1 arc second to here once GMT 6.1.0 comes out.

Reply:
`60m` is removed, and more (e.g., 20m, 15m) are added.